### PR TITLE
Add instruction section.

### PR DESCRIPTION
**Pull request by tblanshard**
Add a small box on the first page detailing how to set up all the required parts/ permissions.

@andreeanica16 @iht @juliocc - thank you :)

### CODE REVIEW

**Review by iht — DISMISSED** [review]
LGTM

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -314,6 +314,7 @@ function getFailedJobsCost(aggregated) {
     jobData.push(category);
     for (costs in aggregated[category]) {
       if (aggregated[category][costs].state == 'JOB_STATE_FAILED') {
+        console.log(aggregated[category][costs].price);
```
Comment (andreeanica16) [inline src/main/webapp/script.js:364]: I think you should delete this :D

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -361,6 +362,8 @@ function transpose(array) {
 }
 
 function getDailyView(aggregated) {
+
+  console.log(aggregated);
```
Comment (andreeanica16) [inline src/main/webapp/script.js:413]: And these lines as well :D

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -405,7 +408,8 @@ function getDailyView(aggregated) {
   }
 
   data = transpose(data);
-  drawLineGraph(data, 'Cost Prediction On Daily Scale', 'costPrediction-container');  
+  console.log(data);
```
Comment (andreeanica16) [inline src/main/webapp/script.js:458]: And this line

**Review by andreeanica16 — CHANGES_REQUESTED** [review]

**Review by andreeanica16 — APPROVED** [review]

**Pull request merged**